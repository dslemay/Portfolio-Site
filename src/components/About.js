import React from 'react';
import Hero from './Hero';
import AboutCards from './AboutCards';

const { Fragment } = React;

const About = () => (
  <Fragment>
    <Hero />
    <AboutCards />
  </Fragment>
);

export default About;
