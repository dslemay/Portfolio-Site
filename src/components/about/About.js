import React from 'react';
import Hero from './Hero';
import AboutCards from './AboutCards';
import Blurb from './Blurb';

const { Fragment } = React;

const About = () => (
  <Fragment>
    <Hero />
    <AboutCards />
    <Blurb />
  </Fragment>
);

export default About;
