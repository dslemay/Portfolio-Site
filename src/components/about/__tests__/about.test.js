import React from 'react';
import { cleanup, render } from 'react-testing-library';

import About from '..';

afterEach(cleanup);

describe('<About />', () => {
  it('renders correctly', () => {
    const { container, getByTestId, getByText } = render(<About />);

    expect(container).toBeTruthy();
    expect(getByTestId('random-quote')).toBeTruthy();
    expect(getByText('Skills')).toBeTruthy();
  });
});
