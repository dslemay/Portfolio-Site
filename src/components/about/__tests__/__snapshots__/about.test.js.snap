// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders correctly 1`] = `
.emotion-1 {
  background-image: url(process:function process(src,filename,config,options) { return 'module.exports = ' + JSON.stringify(path.basename(filename)) + ';';
}

;);background-repeat:no-repeat;background-position:center center;background-attachment:fixed;background-size:cover;height:400px;position:relative;width:100%;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-box-pack:center;-webkit-justify-content:center;-ms-flex-pack:center;justify-content:center;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;z-index:5;}.emotion-1:before {
  background: rgba(230,216,175,0.8);
  position: absolute;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  content: '';
}

.emotion-0 {
  width: 50%;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  text-align: center;
  z-index: 3;
}

.emotion-0 h1 {
  margin-bottom: 0.6rem;
  font-weight: bold;
}

.emotion-0 hr {
  background-color: #2D3B7F;
  margin-bottom: 0.6rem;
  height: 4px;
  border-radius: 5px;
}

.emotion-8 {
  margin: 0 auto;
  margin-bottom: 0.5rem;
  display: block;
}

.emotion-14 {
  grid-area: 2 / 1 / 3 / -1;
}

@media (min-width:700px) {
  .emotion-14 {
    grid-area: 2 / 1 / 3 / span 3;
  }
}

@media (min-width:1000px) {
  .emotion-14 {
    grid-area: 3 / 1 / 4 / span 4;
  }
}

.emotion-7 {
  grid-area: 3 / 1 / 4 / -1;
}

@media (min-width:700px) {
  .emotion-7 {
    grid-column: 2 / span 4;
  }
}

@media (min-width:1000px) {
  .emotion-7 {
    grid-area: 2 / 3 / 3 / 5;
  }
}

.emotion-4 {
  margin-left: 1rem;
}

.emotion-15 {
  width: 80%;
  margin: 0 auto;
  margin-top: 2rem;
  margin-bottom: 2rem;
  display: grid;
  grid-template: repeat(3,auto) / repeat(6,1fr);
  grid-gap: 20px;
}

@media (max-width:450px) {
  .emotion-15 {
    width: 90%;
  }
}

@media (max-width:700px) {
  .emotion-15 {
    grid-gap: 10px;
  }
}

.emotion-2 {
  margin: 0 auto;
  border-radius: 50%;
  max-width: 150px;
  max-height: 150px;
  grid-area: 1 / 1 / 2 / 3;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
}

@media (min-width:700px) {
  .emotion-2 {
    max-width: 175px;
    max-height: 175px;
  }
}

@media (min-width:1000px) {
  .emotion-2 {
    max-width: 250px;
    max-height: 250px;
  }
}

.emotion-3 {
  font-size: 3rem;
  justify-self: center;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  grid-area: 1 / 4 / 2 / span 3;
  color: #2D3B7F;
}

@media (min-width:700px) {
  .emotion-3 {
    grid-column: 5 / span 2;
  }
}

@media (min-width:1000px) {
  .emotion-3 {
    grid-column: 3 / span 2;
    font-size: 4rem;
  }
}

.emotion-6 {
  margin-bottom: 1.5rem;
  border: 1px solid #ccc;
  padding: 0.5rem 0.25rem;
  box-shadow: 3px 3px 5px #656E9A;
}

.emotion-5 {
  position: relative;
  padding: 1rem 1rem 0rem 2rem;
}

.emotion-5::before {
  content: '\\201C';
  font-family: Georgia;
  font-style: bold;
  font-size: 4rem;
  color: #9A8956;
  position: absolute;
  top: -1.4rem;
  left: 0.1rem;
}

.emotion-5::after {
  content: '\\201D';
  font-family: Georgia;
  font-style: bold;
  font-size: 4rem;
  color: #9A8956;
  position: absolute;
  bottom: -3.6rem;
  right: 0.1rem;
}

.emotion-13 {
  grid-area: 4 / 1 / 5 / -1;
}

@media (min-width:700px) {
  .emotion-13 {
    grid-area: 2 / 4 / 3 / -1;
  }
}

@media (min-width:1000px) {
  .emotion-13 {
    grid-area: 1 /5 / -1 / -1;
  }
}

.emotion-9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-9 svg,
.emotion-9 h1 {
  margin: 0;
}

.emotion-10 {
  height: 0.2rem;
  margin-bottom: 0.7rem;
  background-color: #E6D8AF;
}

.emotion-11 {
  padding: 0 2rem;
}

.emotion-11 li {
  margin-bottom: 0.4rem;
  line-height: 1.3;
}

<section
  id="about"
>
  <div
    class="emotion-1"
  >
    <div
      class="emotion-0"
    >
      <h1>
        Daniel Lemay
      </h1>
      <hr />
      <h1>
        Full Stack JavaScript Developer
      </h1>
    </div>
  </div>
  <div
    name="about"
  />
  <div
    class="emotion-15"
  >
    <img
      alt="Profile image of Daniel Lemay"
      class="emotion-2"
      src="[object Object]"
    />
    <h1
      class="emotion-3"
    >
      About
    </h1>
    <div
      class="emotion-7"
    >
      <div
        class="emotion-6"
      >
        <div
          class="emotion-5"
        >
          <div>
            "One option is no option. Two options is a dilemma. Three options is a choice."
          </div>
          <div
            class="emotion-4"
          >
            ~Anthony Robbins
          </div>
        </div>
      </div>
    </div>
    <div
      class="emotion-13"
    >
      <div
        class="emotion-6"
      >
        <div
          class="emotion-9"
        >
          <svg
            class="emotion-8"
            fill="currentColor"
            height="80"
            preserveAspectRatio="xMidYMid meet"
            style="vertical-align: middle; color: rgb(45, 59, 127);"
            viewBox="0 0 40 40"
            width="80"
          >
            <g>
              <path
                d="m12.8 30.4l-1 1.1q-0.2 0.2-0.5 0.2t-0.5-0.2l-9.7-9.7q-0.2-0.2-0.2-0.5t0.2-0.4l9.7-9.7q0.2-0.2 0.5-0.2t0.5 0.2l1 1q0.2 0.2 0.2 0.5t-0.2 0.5l-8.1 8.1 8.1 8.2q0.2 0.2 0.2 0.5t-0.2 0.4z m12.3-22.1l-7.8 26.8q0 0.3-0.3 0.4t-0.5 0.1l-1.3-0.4q-0.2-0.1-0.4-0.3t0-0.5l7.7-26.8q0.1-0.3 0.4-0.4t0.4-0.1l1.3 0.4q0.3 0 0.4 0.3t0.1 0.5z m13.6 13.5l-9.7 9.7q-0.2 0.2-0.4 0.2t-0.5-0.2l-1-1.1q-0.3-0.2-0.3-0.4t0.3-0.5l8.1-8.2-8.1-8.1q-0.3-0.2-0.3-0.5t0.3-0.5l1-1q0.2-0.2 0.5-0.2t0.4 0.2l9.7 9.7q0.2 0.2 0.2 0.4t-0.2 0.5z"
              />
            </g>
          </svg>
          <h1>
            Skills
          </h1>
        </div>
        <hr
          class="emotion-10"
        />
        <ul
          class="emotion-11"
        >
          <li>
            Responsive Web Design
          </li>
          <li>
            Cutting edge web technologies including React, Redux, CSS Grid
          </li>
          <li>
            Backend experience with APIs, Node, Express, and MongoDB
          </li>
          <li>
            Experience working in large and diverse teams
          </li>
          <li>
            Clear Communication and direction to reach end goals
          </li>
        </ul>
      </div>
    </div>
    <div
      class="emotion-14"
    >
      <p>
        <strong>
          My name is Daniel Lemay, and I am a full stack JavaScript developer specializing in React.
        </strong>
         
        I focus on building responsive and accessible front end applications using modern web technologies that are a pleasure to use. I pay attention to the details and work well with numerous stakeholders to complete projects. Take a look at some of my projects below and to contact me about available opportunities.
      </p>
    </div>
  </div>
</section>
`;
